specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/apimpolicyfragments.json

```diff
@@ -0,0 +1,343 @@
+            "$ref": "./apimanagement.json#/parameters/PolicyFragmentIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The specified policy fragment exists and the current entity state version is present in the ETag header.",
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "PolicyFragment"
+        ],
+        "operationId": "PolicyFragment_Get",
+        "description": "Get the policy fragment definition of the Api Management service.",
+        "x-ms-examples": {
+          "ApiManagementGetPolicyFragment": {
+            "$ref": "./examples/ApiManagementGetPolicyFragment.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/PolicyFragmentIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Get the policy fragment definition of the Api Management service.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/PolicyFragmentContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PolicyFragment"
+        ],
+        "operationId": "PolicyFragment_CreateOrUpdate",
+        "description": "Creates or updates the policy fragment configuration of the Api Management service.",
```

Comment:
ISn't this creating or updating the policy fragment - not the configuration.  Description is confusing (I also see the same problem in the description throughout - description goes in the customer facing documentation, so please be descriptive.)

Reply:
I updated most of the descriptions in this document to make them clearer or more concise. Please review again and comment on those that you don't think are clear. Thanks.